### PR TITLE
Bug 1127955 - converted text to aria-label

### PR DESCRIPTION
**Pull request by jobara**
Converted the past-other-month and future-other-month to be aria-labels instead of text nodes.

https://bugzilla.mozilla.org/show_bug.cgi?id=1127955

### CODE REVIEW

**Comment by try-server-hook** [comment]
[jobara Justin Obara (jobara)](https://github.com/jobara) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b344cfd6e5b3)

File: apps/calendar/locales/calendar.en-US.properties [hunk 1]
```diff
@@ -269,8 +269,8 @@ busy[few] = {{n}} events
 busy[many] = {{n}} events
 busy[other] = {{n}} events
 present.ariaLabel = Today
-past-other-month = Other month
-future-other-month = Other month
+past-other-month.ariaLabel = Other month
```
Comment (yzen) [inline apps/calendar/locales/calendar.en-US.properties]: Here and below: need to rename the key to something new (l10n requirement for updating strings).

**Comment by try-server-hook** [comment]
[jobara Justin Obara (jobara)](https://github.com/jobara) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c4e6721c3c99)

**Comment by try-server-hook** [comment]
[jobara Justin Obara (jobara)](https://github.com/jobara) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=26876f02cbc9)

**Pull request merged**